core/src/main/java/com/novoda/noplayer/PlayerView.java

```diff
@@ -19,4 +21,57 @@
     void hideSubtitles();
 
     void setSubtitleCue(TextCues textCues);
+
+    class SurfaceContainer {
```

Comment:
not sure about the name of this thing, also if it should be inside the interface

Reply:
I tried... adding _Player_ at the beginning makes it not confusing at all, right right